src/main/java/org/team1540/robot2024/Constants.java

```diff
@@ -21,7 +21,7 @@
 public final class Constants {
     public static final boolean IS_COMPETITION_ROBOT = true;
     // Whether to pull PID constants from SmartDashboard
-    private static final boolean tuningMode = false; // TODO: DO NOT SET TO TRUE FOR COMP
+    private static final boolean tuningMode = true; // TODO: DO NOT SET TO TRUE FOR COMP
```

Comment:
Was this set to true the whole time :skull:
```suggestion
    private static final boolean tuningMode = false; // TODO: DO NOT SET TO TRUE FOR COMP
```

Reply:
Why turn it off now though?

Reply:
It would be nice to have consistent constants when FMSed and not FMSed. I guess it's not really a probably it just kinda bugs me.